utils/sectors_handler.py

```diff
@@ -0,0 +1,32 @@
+class SectorsHandler(object):
```

Comment:
This should also be deleted for the moment, even if we are going to eventually implement terrain, "sector" is not the right word for it. In this implementation, I don't know that we are doing terrain negotiation.